lib/matplotlib/mlab.py

```diff
@@ -3824,8 +3817,8 @@ def poly_below(xmin, xs, ys):
       xv, yv = poly_below(0, x, y)
       ax.fill(xv, yv)
     """
-    if ma.isMaskedArray(xs) or ma.isMaskedArray(ys):
-        numpy = ma
+    if isinstance(xs, np.ma.MaskedArray) or isinstance(ys, np.ma.MaskedArray):
+        numpy = np.ma
```

Comment:
The problem is that there's another variable called `Nx` in the same function ("size of xs"), which makes things a bit awkward... and `num` sounds too generic, doesn't it?